plugins/catalog-react/src/components/EntityOwnerPicker/EntityOwnerPicker.tsx

```diff
@@ -173,18 +177,25 @@ export const EntityOwnerPicker = (props?: EntityOwnerPickerProps) => {
                 }
                 onClick={event => event.preventDefault()}
                 label={
-                  <Box display="flex" flexWrap="wrap" alignItems="center">
-                    {isGroup ? (
-                      <GroupIcon fontSize="small" />
-                    ) : (
-                      <PersonIcon fontSize="small" />
-                    )}
-                    &nbsp;
-                    {humanizeEntity(
-                      entity,
-                      humanizeEntityRef(entity, { defaultKind: entity.kind }),
-                    )}
-                  </Box>
+                  <Tooltip title={humanizedName}>
```

Comment:
could be good to look into using the presentation API instead

https://backstage.io/docs/reference/plugin-catalog-react.useentitypresentation/

Reply:
Will do! (I also added the screenshot for the previous behaviour) 

Thanks for the comments

Reply:
I run into issues with hooks 
`React Hook "useEntityPresentation" cannot be called inside a callback`. 

If I disable eslint for this line, I get 
`Rendered more hooks than during the previous render`. 

Seems best to keep it as it is for this one

Reply:
Nah that's just how hooks work in React; you can for example make a wrapper component to put the hook in, or similar

Reply:
Thanks for the tip, I changed it to use presentation API now